webpages/settings/index.js

```diff
@@ -639,8 +617,8 @@ let fuse;
   window.onresize = resize;
   resize();
 
-  chrome.management.getSelf((info) => {
-    if (info.installType === "development") vue.devMode = true;
+  getInstallType().then((installType) => {
```

Comment:
It's way more handy if we can expose an `installType` variable. Would also remove the need for all of this code to be asynchronous.

Reply:
it'd be pretty hard to guarantee the call to setup this value would finish before it is requested